Use Jetty v9.4.8

bjorncs opened the pull request:
Upgrade to newest Jetty and add support for empty HTTP header values.
Release notes Jetty: http://dev.eclipse.org/mhonarc/lists/jetty-announce/msg00114.html

gjoranv commented on pom.xml:
```diff
@@ -949,7 +949,7 @@
         <jackson2.version>2.8.3</jackson2.version>
         <jersey2.version>2.23.2</jersey2.version>
-        <jetty.version>9.4.6.v20170531</jetty.version>
+        <jetty.version>9.4.8.v20171121</jetty.version>
```
jetty.version is now defined in container-dependency-versions/pom.xml

bjorncs committed:
Use Jetty v9.4.8

Upgrade to newest Jetty and add support for empty HTTP header values.
Release notes Jetty: http://dev.eclipse.org/mhonarc/lists/jetty-announce/msg00114.html

bjorncs commented:
Let's try this on Monday.

gjoranv commented:
Did you remember the dependency enforcers?

bjorncs commented:
Which one? Doesn't it use the version property from _container-dependency-versions_?

gjoranv commented:
Yes, you're right. I just wanted to make sure you had checked 👍
